src/SilexMtHaml/Lexer.php

```diff
@@ -17,6 +18,12 @@ public function __construct(Environment $environment, \Twig_LexerInterface $lexe
 
     public function tokenize($code, $filename = null)
     {
+        if ($code instanceof Twig_Source) {
+            if ($filename === null)
+                $filename = $code->getName();
```

Comment:
Could you add curly braces around this ? This is required to conform to PSR-2 coding style :)